vcsclient/bitbucketserver.go

```diff
@@ -345,30 +345,12 @@ func (client *BitbucketServerClient) getOpenPullRequests(ctx context.Context, ow
 			return nil, err
```

Comment:
what about bitbucket cloud?

Reply:
We are usually not implementing Bitbucket Cloud for new APIs as we currently don't need it.